shell/platform/tizen/public/flutter_tizen.h

```diff
@@ -91,6 +95,18 @@ FlutterDesktopGetPluginRegistrar(FlutterDesktopEngineRef engine,
 FLUTTER_EXPORT FlutterDesktopMessengerRef
 FlutterDesktopEngineGetMessenger(FlutterDesktopEngineRef engine);
 
+/**
+ * Return backing window for manipulation in host application.
+ *
+ * If run engine on a wearable device, need cast void* to Evas_Objct*,
+ * otherwise cast void* to Ecore_Wl2_Window*.
+ */
+FLUTTER_EXPORT void* FlutterDesktopViewGetWindow(FlutterDesktopViewRef view);
+
+// Returns the view associated with this registrar's engine instance.
+FLUTTER_EXPORT FlutterDesktopViewRef FlutterDesktopPluginRegistrarGetView(
```

Comment:
Do we need the translation from View to Window? It seems just okay to get Window directly from the registrar.

Reply:
Good suggestion, for currently status, we can get window directly form the registrar. Thank you~